packages/wp-story-editor/src/api/pageTemplate.js

```diff
@@ -64,5 +65,5 @@ export function deletePageTemplate(config, id) {
     }),
     data: { force: true },
     method: 'POST',
-  });
+  }); // Response is not being used in core editor.
```

Comment:
Let's make it return `void`, otherwise if it gets used at some point it will be confusing.